content/doc/tutorials/build-a-python-app-with-pyinstaller.adoc

```diff
@@ -62,136 +55,96 @@ Obtain the simple "add" Python application from GitHub, by forking the sample
 repository of the application's source code into your own GitHub account and
 then cloning this fork locally.
 
-. Ensure you are signed in to your GitHub account. If you don't yet have a
-  GitHub account, sign up for a free one on the https://github.com/[GitHub
-  website].
+. Ensure you are signed in to your GitHub account.
+If you don't yet have a GitHub account, sign up for a free one on the https://github.com/[GitHub website].
 . Fork the
-  https://github.com/jenkins-docs/simple-python-pyinstaller-app[`simple-python-pyinstaller-app`]
-  on GitHub into your local GitHub account. If you need help with this process,
-  refer to the https://help.github.com/articles/fork-a-repo/[Fork A Repo]
-  documentation on the GitHub website for more information.
-. Clone your forked `simple-python-pyinstaller-app` repository (on GitHub)
-  locally to your machine. To begin this process, do either of the following
-  (where `<your-username>` is the name of your user account on your operating
-  system):
+  https://github.com/jenkins-docs/simple-python-pyinstaller-app[`simple-python-pyinstaller-app`] on GitHub into your local GitHub account.
+If you need help with this process, refer to the https://help.github.com/articles/fork-a-repo/[Fork A Repo] documentation on the GitHub website for more information.
+. Clone your forked `simple-python-pyinstaller-app` repository (on GitHub)locally to your machine.
+To begin this process, do either of the following (where `<your-username>` is the name of your user account on your operating system):
 ** If you have the GitHub Desktop app installed on your machine:
-.. In GitHub, click the green *Clone or download* button on your forked
-   repository, then *Open in Desktop*.
-.. In GitHub Desktop, before clicking *Clone* on the *Clone a Repository* dialog
-   box, ensure *Local Path* for:
+.. In GitHub, click the green *Clone or download* button on your forked repository, then *Open in Desktop*.
```

Comment:
```suggestion
.. In GitHub, click the green *Code* button on your forked repository, then *Open in Desktop*.
```
Updating text to match current GitHub UI

Reply:
Really? I may have a Chrome plugin or something, because on my browser, "Code" is displayed.
Oooops.